sdk/textanalytics/ai-text-analytics/src/lro/poller.ts

```diff
@@ -20,25 +20,25 @@ export interface AnalysisPollerOptions {
 /**
  * Metadata information for an analysis poller operation.
  */
-export interface JobMetadata {
+export interface OperationMetadata {
   /**
-   * The date and time the job was created.
+   * The date and time the operation was created.
    */
   createdOn?: Date;
   /**
-   * The date and time when the job results will expire on the server.
+   * The date and time when the operation results will expire on the server.
    */
   expiresOn?: Date;
   /**
-   * The job id.
+   * The operation id.
    */
-  jobId?: string;
+  operationId?: string;
   /**
-   * The time the job status was last updated.
+   * The time the operation status was last updated.
    */
-  updatedOn?: Date;
+  lastModifiedOn?: Date;
   /**
-   * The current status of the job.
+   * The current status of the operation.
    */
   status?: State;
```

Comment:
is `State` too generic? I'm not sure if this is a problem/concern in JS.
In .NET we renamed the type to `TextAnalyticsOperationStatus`

Reply:
Fixed.